frontend/src/components/archive/ExportArchiveButton.tsx

```
@@ -0,0 +1,116 @@
+        className={`
+          inline-flex items-center px-4 py-2 rounded-lg
+          text-sm font-medium transition-colors
+          ${
+            isExporting
+              ? 'bg-gray-300 text-gray-500 cursor-not-allowed'
+              : 'bg-blue-500 text-white hover:bg-blue-600'
+          }
+        `}
+        aria-label="Export archive to CSV"
+        aria-busy={isExporting}
+      >
```

Biome (2.1.2)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)